.env.example

```diff
@@ -9,8 +9,7 @@ AWS_REGION='key_AWS_REGION'
 AWS_S3_BUCKET='action-files.parabol.co'
 AWS_SECRET_ACCESS_KEY='key_AWS_SECRET_ACCESS_KEY'
 CDN_BASE_URL='key_CDN_BASE_URL'
-# Uncomment when testing locally
-# CI='true'
+CI='true'
```

Comment:
+1 I think there is no harm in enabling this flag locally, but naming is not ideal if we always enable it for development and testing.